policyengine_us/variables/gov/states/co/ccap/entry/co_ccap_hhs_fpg_eligible.py

```diff
@@ -0,0 +1,36 @@
+from policyengine_us.model_api import *
+
+
+class co_ccap_hhs_fpg_eligible(Variable):
+    value_type = bool
+    entity = TaxUnit
+    label = "Eligible for the Colorado Child Care Assistance Program through the household size federal poverty guidelines"
+    reference = (
+        "https://www.sos.state.co.us/CCR/GenerateRulePdf.do?ruleVersionId=11042&fileName=8%20CCR%201403-1#page=19",
+        "https://docs.google.com/spreadsheets/d/1WzobLnLoxGbN_JfTuw3jUCZV5N7IA_0uvwEkIoMt3Wk/edit#gid=1350122430",
+    )
+    definition_period = MONTH
+    # defined_for = StateCode.CO
+
+    def formula(tax_unit, period, parameters):
+        if tax_unit.household("state_code_str", period.this_year) != "CO":
+            return
+        spm_unit = tax_unit.spm_unit
+        monthly_agi = np.round(
+            tax_unit("adjusted_gross_income", period.this_year)
+            / MONTHS_IN_YEAR,
+            2,
+        )
+        year = period.start.year
+        month = period.start.month
+        if month >= 10:
```

Comment:
what's the best way to avoid duplicating these four lines of code @nikhilwoodruff? see remainder of pr

Reply:
per our conversation, we'll keep the duplication for now. but please make `10` a policy parameter.